### DISCUSSION TITLE
How much do we like "Discussions"?

### DISCUSSION THREAD

**Discussion by bohandley** [grafana/grafana#58922] [General]
What has your experience been with discussions? 

Do you like them, why or why not? 

For context, the issue which moved us to using discussions: https://github.com/grafana/grafana/discussions/36963

**Comment by taylorchu** [comment]
most of the discussion topics have no replies, like mine: https://github.com/grafana/grafana/discussions/59343. I would like someone from grafana to triage more frequently. Otherwise, it is uncertain whether something will be done in the near future; should I wait for grafana team to fix it, or I just go ahead and fix it myself?

**Comment by DmitrySidorow** [comment]
most discussions are abandoned or remain unanswered

**Comment by bobemoe** [comment]
I always search issues but quite often forget the existance of discussions. Only ended up here after going to open issue but as its a feature request got redirected here. As it seems not to get much attention I'm tempted to open it as an issue :/

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
